doc/index.rst

```diff
@@ -371,6 +376,19 @@ Output Commands
    status for the given output(s); a list of one or more names or
    numbers is required.
 
+:command:`moveoutput <output # or name>` - Moves the given output 
+   to the current partition.
+
+Partition Commands
+^^^^^^^^^^^^^^^^^^
+
+:command:`partitions` - Lists all available partitions
+
+:command:`makepart <name>...` - Creates new partitions
```

Comment:
Is it really useful that mpc can create multiple partitions with one command? Usually, a new partition needs to be set up (by moving outputs), which requires additional mpc calls anyway; so not much is saved here.
If you want to reduce overhead, you better have a combined command which creates a partition and sets it up.

Reply:
I don't feel strongly, but I like how `mpc add` lets me specify multiple things and thought it'd be nice here as well.

Doing the setup in one command seems like it'd require more complex argument parsing than mpc already has, and I didn't feel like hacking on that.  So I think either leave as-is, or change it to only allow creating/deleting one at a time.  Let me know which you'd like.